### ISSUE TITLE
Feature Request: Multiple quotes in a single API call

### ISSUE THREAD

**Issue by joelsshook** [finnhubio/Finnhub-API#380]
I've seen a couple posts here asking how someone would get several quotes from a single API call, such as:
https://finnhub.io/api/v1/quote?symbol=AAPL&symbol=AMZN&symbol=F
or
https://finnhub.io/api/v1/quote?symbol=AAPL,AMZN,F
or 
https://finnhub.io/api/v1/quote?symbol[]=AAPL&symbol[]=AMZN&symbol[]=F

All the replies I saw were just "not possible" so I would like to request that it be made possible.

Right now, in order to ensure I respect rate limits on my free account, I have my program sleep for 1 second before each api call and this works. However it could work better if I could query all of them at once, or in batches. Even doing it 2 at a time would be a big gain for me, since right now it takes about 6 minutes to run my fairly simple script.

Thank you

**Comment by oscerd** [comment]
+1.

**Comment by boherna** [comment]
Yes, getting quotes for multiple symbols is a must and other APIs already support it. I'd like an endpoint like:
https://finnhub.io/api/v1/quote?symbols=[AAPL,DIS,XL,AI]

**Comment by drddavi** [comment]
+1

**Comment by MonsieurZbanowanYY** [comment]
Still waiting...

**Comment by kalpeshjethva18** [comment]
waiting...

**Comment by strumpet** [comment]
+1

**Comment by enes-ay** [comment]
+1
